code/modules/power/singularity/containment_field.dm

```diff
@@ -158,5 +158,8 @@
 	has_shocked = TRUE
 	do_sparks(5, TRUE, considered_atom.loc)
 	var/atom/target = get_edge_target_turf(considered_atom, get_dir(src, get_step_away(considered_atom, src)))
-	considered_atom.throw_at(target, 200, 4)
+	if(isliving(considered_atom))
+		to_chat(considered_atom, span_userdanger("The field repels you with tremendous force!"))
+	playsound(src, 'sound/effects/gravhit.ogg', 50, TRUE)
+	considered_atom.throw_at(target, rand(6, 12), 4)
```

Comment:
```suggestion
	considered_atom.throw_at(target, 200, 4)
```
a distance of 200 is fine, imo. a short, rng-based distance like the one you proposed is just gonna make it impossible to ping pong people up and down a hallway, which is funny.

Reply:
the stun length reduction is definitely good, though